unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/GLTF/Tests/AnyAssetPromiseShould.cs

```diff
@@ -191,5 +190,38 @@ public IEnumerator NotTryToLoadAfterForget()
 
             keeper.Cleanup();
         }
+
+        [UnityTest]
+        public IEnumerator WaitForGLTFtoBeLoadedBeforeDestroyingIt()
```

Comment:
Just a camelcase nitpick:

```suggestion
        public IEnumerator WaitForGltfToBeLoadedBeforeDestroyingIt()
```
or
```suggestion
        public IEnumerator WaitForGLTFToBeLoadedBeforeDestroyingIt()
```

Reply:
I like the first suggestion :)